apps/dialtone-documentation/docs/.vuepress/baseComponents/CodeExampleTabs.vue

```diff
@@ -71,7 +73,7 @@ const props = defineProps({
    */
   htmlCode: {
     type: [String, Function],
-    required: true,
+    default: null,
```

Comment:
For certain cases it doesn't add much value to show the html, particularly for complex components. Here for example I am showing the html for the dropdown in a couple of examples, but then when showing the variant for a single Vue prop that only changes functionality (`openOnContext`) I don't think the html is needed.

Reply:
Yep I agree with this